spec/docs/participant_declarations_spec.rb

```diff
@@ -20,37 +21,104 @@
           "schema": {
             "type": "object",
             "properties": {
-              "name": "id",
-              "type": "string",
+              "participant_id": {
+                "type": "string",
+              },
+              "declaration_type": {
+                "enum": %w[Start],
```

Comment:
Should these be capitalised? 😬

Reply:
No they shouldn't. We also changed the name during the session on Friday. They need to be updated.